packages/vscode-extension/package.json

```diff
@@ -23,7 +23,7 @@
 		"**/*.vsix"
 	],
 	"engines": {
-		"vscode": "^1.57.0"
+		"vscode": "^1.63.1"
```

Comment:
If we change this line we should also edit the `/packages/vscode-extension/README.md` that contains a related reference.
`VS Code version 1.57 or higher`.

Reply:
Done. Good catch